apps/browser-extension-wallet/src/views/browser-view/features/shared-wallet/create-flow/SharedWalletCreationStore.tsx

```diff
@@ -124,7 +134,13 @@ const makeInitialState = (activeWalletName: string): State => ({
 
 const getInitialCoSignerValue = (): CoSigner => ({ id: uuid(), keys: '', name: '' });
 
-const makeStateMachine = ({ navigateHome }: { navigateHome: () => void }): StateMachine => ({
+const makeStateMachine = ({
+  navigateHome,
+  navigateToStart
+}: {
+  navigateHome: () => void;
```

Comment:
IMO Home might be understood differently. If you agree consider renaming to something more explicit.
```suggestion
  navigateToAssetsPage: () => void;
```

Reply:
If we decide some day that home !== assets, then we would have to change this. :)